modules/root/root_api.rb

```diff
@@ -25,4 +27,19 @@ class Proxy::RootApi < Sinatra::Base
       log_halt 400, e
     end
   end
+
+  get "/events" do
+    begin
+      content_type :json
+
+      events = []
+      from_serial = params[:from_serial].to_i rescue 0
```

Comment:
also, why is it called from_serial? its a timestamp isnt it? so it should be called from_timestamp?